src/main/java/com/gamingmesh/jobs/listeners/JobsListener.java

```diff
@@ -362,6 +363,13 @@ public void onCropGrown(final BlockGrowEvent event) {
 	}
     }
 
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void onTreeGrown(final StructureGrowEvent event) {
+	if (Jobs.getGCManager().canPerformActionInWorld(event.getBlocks().get(0).getWorld())) {
+	    plugin.getServer().getScheduler().scheduleSyncDelayedTask(plugin, () -> event.getBlocks().forEach( blockState -> Jobs.getBpManager().remove(blockState.getBlock())), 1L);

```

Comment:
`runTaskLater` and remove whitespace after `forEach`

Reply:
oh yes, I just used `scheduleSyncDelayedTask` since this was used in the `onCropGrown` listener. So I thought it might have been that way to make it backward compatible.